src/app.jsx

```diff
@@ -101,7 +106,37 @@ export const App = () => {
                 .finally(() => setLoadingResources(false));
     }, []);
 
-    if ((superuser.allowed && systemSocketInactive) || loadingResources) {
+    useEffect(() => {
+        async function checkVirtualization() {
+            const hardwareVirtCheck = await cockpit.script("virt-host-validate | grep 'Checking for hardware virtualization'");
+
+            setVirtualizationEnabled(hardwareVirtCheck.includes('PASS'));
+        }
+
+        checkVirtualization();
+    }, []);
+
+    if (!virtualizationEnabled && !emptyStateIgnored) {
+        return (
+            <EmptyState>
+                <EmptyStateIcon icon={WrenchIcon} />
+                <Title headingLevel="h4" size="lg">
+                    {_("Virtualization support is not available")}
+                </Title>
+                <EmptyStateBody>
+                    <Text>{_("Enable virtualization support in BIOS/EFI settings.")}</Text>
+                    <Text>
+                        {_(`Instructions vary per manufacturer, but usually involves pressing a hot key such as ESC, F1, F12, or Del during boot.
+                        In the settings screen, look for "virtualization", "VM", "VMX', "SVM", "VTX", "VTD". It will be different on every computer.
+                        Enable any of these options. Consult your computer's manual for details.`)}
+                    </Text>
+                </EmptyStateBody>
+                <EmptyStatePrimary>
+                    <Button variant="link" onClick={() => setEmptyStateIgnored(true)}>{_("Ignore")}</Button>
```

Comment:
Since we don't use any cookie and we store that in react state that 'ignore' wil last only till the user refreshes the page. I think it's fine as first pass, but we probably want to either use a cookie for remembering the option or reconsider the design and not hide the whole app behind this empty state. But fine for follow up.

Reply:
We could also use localStorage.

Reply:
Thanks for the feedback. I implemented saving the preference in localStorage.